mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1943,6 +1943,11 @@ def LLVM_CallIntrinsicOp
     attr-dict
   }];
 
+  let builders = [
+    OpBuilder<(ins "Type": $results, "StringAttr":$intrin, "ValueRange":$args)>,
+    OpBuilder<(ins "TypeRange": $results, "StringAttr":$intrin, "ValueRange":$args, "FastmathFlagsAttr":$fastMathFlags)>
+    ];
```

Comment:
Maybe we could also include builders for intrinsics that do not have a result type?

```
OpBuilder<(ins "StringAttr":$intrin, "ValueRange":$args)>
OpBuilder<(ins "StringAttr":$intrin, "ValueRange":$args, "FastmathFlagsAttr":$fastMathFlags)>
```

Reply:
Thanks for the review.
I've added those in. Do you know how I could confirm they work? I don't really have a use cause so I was just guessing.

Reply:
Another approach might be adding a unit test under `mlir/unittests/Dialect/LLVMIR`. But it seems like few people ever do this ... Anyway this builder is fairly simple and I believe it should be OK if no reviewers have problems about it.